### PR TITLE
nicotine+: update to 2.1.2

### PR DESCRIPTION
**Pull request by st3r4g**
Nicotine+ switched from `mutagen` to `pytaglib`. The latter is not in the repos yet.
I am not sure whether `GPL-3.0-only` or `GPL-3.0-or-later` is right for `pytaglib`

Tested on x86_64-musl

### CODE REVIEW

**Comment by sgn** [comment]
> I am not sure whether `GPL-3.0-only` or `GPL-3.0-or-later` is right for `pytaglib`

I'll put my money on `GPL-3.0-only`

**Comment by ericonr** [comment]
https://pypi.org/project/pytaglib/ says it's `GPL-3.0.-or-later`.

**Comment by st3r4g** [comment]
Good catch, changed

**Comment by mathiascode** [comment]
We've released Nicotine+ 2.1.2 earlier today. https://nicotine-plus.github.io/nicotine-plus/NEWS.html

**Comment by st3r4g** [comment]
Thanks, updated

File: srcpkgs/python3-pytaglib/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-pytaglib'
+pkgname=python3-pytaglib
+version=1.4.6
+revision=1
+wrksrc="pytaglib-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel taglib-devel"
+depends="taglib"
```
Comment (ahesford) [inline srcpkgs/python3-pytaglib/template:9]: Drop `taglib`, it is automatically detected.

**Review by ahesford — CHANGES_REQUESTED** [review]

File: srcpkgs/python3-pytaglib/template (hunk 1 above)
Comment (st3r4g) [reply srcpkgs/python3-pytaglib/template]: wait a sec I messed up

Comment (st3r4g) [reply srcpkgs/python3-pytaglib/template]: ok fixed

**Pull request closed without merging**